src/subscription_manager/managercli.py

```diff
@@ -1824,6 +1824,18 @@ def _do_command(self):
         if self.options.pool or self.options.file:
             self.auto_attach = False
 
+        # Do not try to do auto-attach, when simple content access mode is used
+        # BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1826300
+        if self.auto_attach is True:
+            if is_simple_content_access(uep=self.cp, identity=self.identity):
+                owner = get_current_owner(self.cp, self.identity)
```

Comment:
The method `get_current_owner()` was introduced in this PR: #2155 and it is not possible to use in `subscription-manager-1.24` branch. I updated this PR to communicate with candlepin server directly.